pkg/registry/core/service/storage/rest_test.go

```diff
@@ -2280,3 +2312,75 @@ func TestUpdateNodePorts(t *testing.T) {
 		}
 	}
 }
+
+func TestServiceRegistryCreateWithNetworkOfExternalIPAM(t *testing.T) {
+	storage, registry, server := NewTestREST(t, nil)
+	storage.networks = &testNetworkGetter{ipamType: "External"}
+	defer server.Terminate(t)
+
+	svc := &api.Service{
```

Comment:
Add Tenant/Namespace in ObjectMeta.

Reply:
will add the default tenant/ns explicitly.